### PR TITLE
feat: Implement Crashlytics

### PR DESCRIPTION
**Pull request by dilpreet2028**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -55,6 +55,9 @@
         <service android:name=".ui.widgets.ChargeWidgetService"
                  android:permission="android.permission.BIND_REMOTEVIEWS" />
 
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="8dbc177cd5391a21947da95f2aa9208f33cd45f4" />
```
Comment (therajanmaurya) [inline app/src/main/AndroidManifest.xml]: Put this key to https://github.com/openMF/self-service-app/blob/development/app/src/debug/res/values/google_maps_api.xml and Make the file name `google_maps_api.xml` -> `api_keys.xml`, In both debug and release.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by dilpreet2028** [comment]
@therajanmaurya on placing the keys in debug and release , gradle was throwing error on build 

`Error:Execution failed for task ':app:fabricGenerateResourcesDebug'. Crashlytics Developer Tools error.`

On searching for this I came to know that fabric does not allow string references and was working fine on placing raw string in manifest so I followed their official guide for placing the api key 
https://docs.fabric.io/android/fabric/settings/working-in-teams.html#android-projects

**Comment by therajanmaurya** [comment]
ok, You have to change release api key file name too.

**Comment by dilpreet2028** [comment]
@therajanmaurya ,if I am getting you right then, both debug and release xml are already named as api_key.xml , please have a look

**Pull request merged**